index_release_test.go

```diff
@@ -36,6 +36,16 @@ func Test_buildReleases(t *testing.T) {
 				{
 					Active: true,
 					Authorities: []Authority{
+						{
+							Endpoint: urlMustParse("http://app-operator:8000/"),
+							Name:     "app-operator",
+							Version:  "0.1.0",
+						},
+						{
+							Endpoint: urlMustParse("http://appcatalog-operator:8000/"),
```

Comment:
I don't get the `appcatalog-operator`? We're not going to create this operator. app-operator needs to have 2 controllers.

If you need another operator for the test then picking an existing operator is less confusing IMO.

Reply:
oops, I will change the name as `app-controller` and `appcatalog-controller` then.